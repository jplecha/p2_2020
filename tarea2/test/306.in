insertarAlFinal (5,3.8)
insertarAlFinal (3,4.9)
insertarAlFinal (9,3.9)
insertarAlFinal (5,5.2)
insertarAlFinal (5,1.3)
insertarAlFinal (3,3.7)
insertarAlFinal (4,5.2)
insertarAlFinal (6,6.9)
insertarAlFinal (7,4.6)
esLocalizador
finalCadena
finalCadena
imprimirCadena
esLocalizador
esLocalizador
finalCadena
insertarAlFinal (3,7.8)
finalCadena
insertarAlFinal (6,6.7)
finalCadena
ordenar
insertarAntes (6,3.8)
insertarSegmentoDespues (9,5.7) (4,3.7) (6,3.3) (6,1.6) (9,5.6) (3,9.9) (7,9.1) (6,1.7) (9,5.9) (4,1.8)
copiarSegmento 12 12
estaOrdenadaPorNaturales
