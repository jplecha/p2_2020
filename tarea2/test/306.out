principal: principal.cpp:327: int main(): Assertion `! hayNatsRepetidos(cad)' failed.
1>Insertado al final.
2>Insertado al final.
3>Insertado al final.
4>Insertado al final.
5>Insertado al final.
6>Insertado al final.
7>Insertado al final.
8>Insertado al final.
9>Insertado al final.
10>loc no válido.
11>loc al final.
12>loc al final.
13>(5,3.80)(3,4.90)(9,3.90)(5,5.20)(5,1.30)(3,3.70)(4,5.20)(6,6.90)(7,4.60)
14>loc válido.
15>loc válido.
16>loc al final.
17>Insertado al final.
18>loc al final.
19>Insertado al final.
20>loc al final.
21>timeout: the monitored command dumped core
